### ISSUE TITLE
enable file attachments on mapping wiki

### ISSUE THREAD

**Issue by VladimirAlexiev** [dbpedia/mappings-tracker#43]
@alexandrutodor:
I want to attach a file (Iana-lang-tags.xlsx) to http://mappings.dbpedia.org/index.php/What%27s_in_a_Name#Language-specific_Names so people can more easily find lang tags for #15 (eg Moldovan is "mo" and Gagauz is "gag")

I made a redlink using the toolbar, but when I go to it http://mappings.dbpedia.org/index.php?title=File:Iana-lang-tags.xlsx&action=edit&redlink=1 I don't see an Upload button.

@jplu, @jimregan: if you want the file I'll put it on Dropbox. Or you could make it yourself using the perl script referenced in the page above.

**Comment by jimkont** [comment]
Why not use google docs?

**Comment by VladimirAlexiev** [comment]
You're right, put on dropbox. Too often I don't consider the easiest workaround.
